uber/site_sections/groups.py

```diff
@@ -69,10 +69,8 @@ def unapprove(self, session, id, action, email):
             send_email(MARKETPLACE_EMAIL, group.email, subject, email, model = group)
         if action == 'waitlisted':
             group.status = WAITLISTED
-        #else:
-            #for attendee in group.attendees:
-                #session.delete(attendee)
-            #session.delete(group)
+        else:
+            self.delete()
```

Comment:
I'm pretty sure this will raise an exception, unless I'm missing something, since the ``delete`` definition below takes several required arguments.